packages/imperative/src/config/cmd/init/init.handler.ts

```diff
@@ -279,9 +185,9 @@ export default class InitHandler implements ICommandHandler {
         // coerce to correct type
         if (propValue && propValue.trim().length > 0) {
             if (propValue === "true")
-                propValue = true;
+                return true;
             if (propValue === "false")
-                propValue = false;
+                return false;
             if (!isNaN(propValue) && !isNaN(parseFloat(propValue)))
                 propValue = parseInt(propValue, 10);
         }
```

Comment:
We probably should change this, otherwise the `return propValue || null` below would prevent us being able to store "0"

Reply:
> We probably should change this, otherwise the `return propValue || null` below might prevent us being able to store the number `0`

This isn't true since we discovered (in comment below) that `0 || null` evaluates to 0 not null. Still going to add a "return" here though :)